crates/jet1090/src/cli.rs

```diff
@@ -1,15 +1,27 @@
 use std::str::FromStr;
 
+use radarcape::Address;
 use rs1090::decode::{cpr::Position, TimedMessage};
 use rs1090::prelude::*;
 use serde::{Deserialize, Serialize};
 use tokio::sync::mpsc::Sender;
+use tracing::error;
+use url::Url;
+
+#[derive(Debug, Clone, Default, PartialEq, Serialize, Deserialize)]
+pub enum Protocol {
```

Comment:
Since `host` and `port` don't make sense for `Rtlsdr`, maybe it should be `Tcp(String)` and `Udp(String)` as well...